internal/service/dashboards_test.go

```diff
@@ -0,0 +1,71 @@
+package service
```

Comment:
currently, all the integration tests are under `test`, unless you have some package level see if you can just add this to the current test.  `dashboard_integration_test.go`